### PR TITLE
ASR validation cases for concrete block

### PR DESCRIPTION
**Pull request by SudiptaBiswas**
Adds more ASR validation cases for concrete blocks.
Refs #89

### CODE REVIEW

**Comment by SudiptaBiswas** [comment]
@bwspenc This is ready for review/merge.

File: assessment/asr_validation/wald2017B/analysis/asr_concrete_block_biaxial_rebar.i [hunk 1]
```diff
@@ -0,0 +1,1070 @@
...
+    block = '2 3'
+    variable = axial_stress
+    property = axial_stress
+  [../]
+[]
+
+[Functions]
+  [./ramp_temp]
+    type = PiecewiseLinear
+    x = ' 0.0 2722893.36 3498624.13 4274354.9 5050085.66 5825816.43 6566286.71 7306756.99 8082487.76 8858218.53 9633949.3 10409680.1 11185410.8 11961141.6
```
Comment (bwspenc) [inline assessment/asr_validation/wald2017B/analysis/asr_concrete_block_biaxial_rebar.i:446]: I think these temperature and humidity functions are shared across multiple cases, aren't they?  If so, I would prefer that we have them defined in csv files that we read in for these functions.

File: assessment/asr_validation/wald2017B/analysis/asr_concrete_block_uniaxial_rebar.i [hunk 2]
```diff
@@ -154,7 +154,7 @@
     type = EqualValueEmbeddedConstraint
     secondary = 2
     primary = 1
-    penalty = 31e6
+    penalty = 31e9
```
Comment (bwspenc) [inline assessment/asr_validation/wald2017B/analysis/asr_concrete_block_uniaxial_rebar.i:157]: We should change these to a number that is simply something like 1e9 so it doesn't look like it's physically meaningful.  Also, I think these work with even higher numbers, maybe 1e12 (at least that's been my experience on similar models).

**Review by bwspenc — COMMENTED** [review]
Just a couple minor things as noted.

Also, I don't understand the naming convention in the input files. what's the difference between '...rebar_config.i' and '...rebar.i', for instance? The names should say something about what makes them unique.

File: assessment/asr_validation/wald2017B/analysis/asr_concrete_block_uniaxial_rebar.i (hunk 2 above)
Comment (SudiptaBiswas) [reply assessment/asr_validation/wald2017B/analysis/asr_concrete_block_uniaxial_rebar.i:157]: I was having convergence issues with higher numbers. This is as high as I could go.

**Comment by SudiptaBiswas** [comment]
'rebar_config' represents the input file used for evaluating the effect of various rebar configuration. I change the mesh in the assessment cases while keeping the input file same. I will think of a better name.

**Comment by bwspenc** [comment]
Looks like the test is failing because documentation is pointing to a file that got changed.

**Comment by SudiptaBiswas** [comment]
I updated the documentation. It's ready for another review.

**Review by bwspenc — APPROVED** [review]

**Pull request merged**